packages/host/app/components/operator-mode/card-error.gts

```diff
@@ -114,6 +114,9 @@ export default class CardErrorComponent extends Component<Signature> {
         background-color: var(--boxel-100);
         box-shadow: 0 1px 0 0 rgba(0 0 0 / 15%);
       }
+      .error-header :deep(.actions) {
+        color: var(--boxel-dark);
+      }
```

Comment:
We can set the color `var(--boxel-error-300)` on only the title (`.card-type-display-name`) and not on the whole header.